src/shared/api.ts

```diff
@@ -1074,6 +1078,58 @@ export const deepSeekModels = {
 	},
 } as const satisfies Record<string, ModelInfo>
 
+// Hugging Face Inference Providers
+// https://huggingface.co/docs/inference-providers/en/index
+export type HuggingFaceModelId = keyof typeof huggingFaceModels
+export const huggingFaceDefaultModelId: HuggingFaceModelId = "moonshotai/Kimi-K2-Instruct"
+export const huggingFaceModels = {
+	"moonshotai/Kimi-K2-Instruct": {
```

Comment:
might be worth including one or two models that use an explicit `:provider` suffix to improve the discoverability and bring more visibility to the underlying partners. what do you think?
```suggestion
	"moonshotai/Kimi-K2-Instruct:groq": {
```